functions/lib/integration/parsers/product-to-ecomplus.js

```diff
@@ -87,9 +87,11 @@ module.exports = (blingProduct, variations, storeId, auth, isNew = true) => new
     cost_price: blingProduct.preco_custo
   }
 
-  if (blingProduct.descricaoComplementar) {
+  if (blingProduct.descricaoComplementar && blingProduct.descricaoCurta) {
     product.short_description = String(blingProduct.descricaoCurta.slice(0, 255))
     product.body_html = String(blingProduct.descricaoComplementar)
+  } else if (blingProduct.descricaoComplementar) {
+    product.body_html = String(blingProduct.descricaoComplementar)
   } else {
```

Comment:
```suggestion
  } else if (blingProduct.descricaoCurta) {
```

Agora não daria erro, só pra evitar o `product.body_html = 'undefined'` ou `product.body_html = 'null'`.